README.md

```diff
@@ -2,21 +2,21 @@
 
-[![](docs/.gitbook/assets/airbyte_white-background%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29.svg)](https://airbyte.io)
+[![](docs/.gitbook/assets/airbyte_white-background%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%282%29.svg)](https://airbyte.io)
 
-### Data integration made simple, secure and reliable.
+### Data integration made simple, secure and extensible.
 
-The new open-source standard to sync data from applications & databases to warehouses.
+The new open-source standard to sync data from applications, APIs & databases to warehouses.
 
-[![](docs/.gitbook/assets/deploy-locally%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29.svg)](https://docs.airbyte.io/deployment/deploying-airbyte/with-docker) [![](docs/.gitbook/assets/deploy-on-aws%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29.svg)](https://docs.airbyte.io/deployment/deploying-airbyte/on-aws-ec2) [![](docs/.gitbook/assets/deploy-on-gcp%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29.svg)](https://docs.airbyte.io/deployment/deploying-airbyte/on-gcp-compute-engine)
+[![](docs/.gitbook/assets/deploy-locally%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%282%29.svg)](https://docs.airbyte.io/deployment/deploying-airbyte/with-docker) [![](docs/.gitbook/assets/deploy-on-aws%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%282%29.svg)](https://docs.airbyte.io/deployment/deploying-airbyte/on-aws-ec2) [![](docs/.gitbook/assets/deploy-on-gcp%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%282%29.svg)](https://docs.airbyte.io/deployment/deploying-airbyte/on-gcp-compute-engine)
 
-![](docs/.gitbook/assets/sources_list%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29.png)
+![](docs/.gitbook/assets/sources_list%20%281%29%20%282%29%20%281%29%20%281%29%20%281%29%20%281%29%20%282%29%20%282%29.png)
 
 Airbyte is on a mission to make data integration pipelines a commodity.
 
 * **Maintenance-free connectors you can use in minutes**. Just authenticate your sources and warehouse, and get connectors that adapt to schema and API changes for you.
-* Designed to **cover the long tail of integrations**, as Airbyte will be active in maintaining the pipelines’ reliability. 
-* **Building new integrations made trivial**. We make it very easy to add new integrations that you need, by offering scheduling and orchestration. 
+* Designed to **cover the long tail of integrations and needs.** Benefit from the community's battle-tested connectors and adapt them to your specific needs. 
+* **Building new integrations made trivial**. We make it very easy to add new integrations that you need, using the language of your choice, by offering scheduling and orchestration. 
```

Comment:
can we make this the second bullet point?

Reply:
Done